man/mvn_build.txt

```diff
@@ -73,6 +73,12 @@ ENVIRONMENT
     behavior is mostly useful when bootstrapping new packages, which
     dependecies have not yet been packaged.
 
+*_without_tests*::
+	Defining this macro to any value is equivalent to *-f* option.
```

Comment:
Formatting and indentation - please use spaces and the same level of indentation as other parts.